kvmagent/kvmagent/plugins/ovn.py

```
@@ -407,6 +453,87 @@ def ovn_del_port(self, req):
+    def ovn_sync_ports(self, req):
+        cmd = jsonobject.loads(req[http.REQUEST_BODY])
+        rsp = kvmagent.AgentResponse()
+
+        logger.debug("cmd: %s: %s" % (cmd, cmd.__dict__))
+        logger.debug("cmd nicMap: %s: %s" % (cmd.nicMap, cmd.nicMap.__dict__))
+        vsctl = ovn.VsCtl()
+
+        localVnics = vsctl.getVnicsByVmUuid(cmd.vmUuid)
+
+        for localVnic in localVnics:
+            if not localVnic in cmd.nicMap.__dict__.keys():
```

Ruff (0.14.0)

Test for membership should be `not in`

Convert to `not in`

(E713)